packages/components/addon/components/hds/form/field/index.js

```diff
@@ -3,7 +3,17 @@ import Component from '@glimmer/component';
 import { guid } from '../utils/guid';
 import { describedBy } from '../utils/describedby';
 
-export default class HdsFormRadioFieldComponent extends Component {
+export default class HdsFormFieldIndexComponent extends Component {
+  /**
+   * Sets the layout of the field
+   *
+   * @param layout
+   * @type {enum}
+   */
+  get layout() {
+    return this.args.layout;
+  }
+
```

Comment:
[suggestion] maybe we should add the different layouts as ENUMs in the components, and add assertions that check that the user passes the right value (in a different ticket/new PR)

(later we can add also tests for this)

Reply:
Good suggestion; pushed a commit to raise a warning if `@layout` is not `vertical` or `flag`